owncloudComLibrary/build.gradle

```diff
@@ -29,11 +29,11 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 34
 
     defaultConfig {
         minSdkVersion 24
-        targetSdkVersion 33
+        targetSdkVersion 34
```

Comment:
Let's use the same variables that we use in the other modules, we also want this one to go in consonancy with the rest of modules

Reply:
Done